RoddyCore/src/de/dkfz/roddy/core/ExecutionContext.groovy

```diff
@@ -545,15 +545,6 @@ class ExecutionContext {
         return new LinkedList<>(errors)
     }
 
-    /**
-     * Execute the stored dataset with the stored analysis.
-     *
```

Comment:
Don't know. I preferred the syntax before

Reply:
Maybe we change things so, that the context has its copy of the workflow and not the analysis. Then the workflow could also store info.

Reply:
I am not a fan of short-circuiting method call chains (and this here is probably already a short one) and would opt for intuitive semantics over shorter code. See my discussion above. 

Note here that the method does not actually do much. The real code already is in the workflow class!

Reply:
The thing is, that context has a lot of shortcut methods, which pull a lot of logic from different places to one place. There's even a design pattern which covers this: Facade.
I know, that it sometimes might be confusing to have the shortcuts but if you get rid of them, you always have call chains. This then makes code look ugly (and sometimes also confusing)